tests/test_bulk_tx_noack/src/main.xc

```diff
@@ -0,0 +1,70 @@
+// Copyright 2016-2021 XMOS LIMITED.
+// This Software is subject to the terms of the XMOS Public Licence: Version 1.
+#include "shared.h"
+
+#ifndef PKT_LENGTH_START
+#define PKT_LENGTH_START   (10)
+#endif
+
+#ifndef PKT_LENGTH_END
+#define PKT_LENGTH_END     (13)
+#endif
+
+#define EP_COUNT_OUT       (6)
+#define EP_COUNT_IN        (6)
+
+/* Endpoint type tables */
+XUD_EpType epTypeTableOut[EP_COUNT_OUT] = {XUD_EPTYPE_CTL,
+                                                XUD_EPTYPE_BUL,
+                                                 XUD_EPTYPE_BUL,
+                                                 XUD_EPTYPE_BUL,
+                                                 XUD_EPTYPE_BUL,
+                                                 XUD_EPTYPE_BUL};
+XUD_EpType epTypeTableIn[EP_COUNT_IN] =   {XUD_EPTYPE_CTL, 
+                                                XUD_EPTYPE_BUL,
+                                                 XUD_EPTYPE_BUL,
+                                                 XUD_EPTYPE_BUL,
+                                                XUD_EPTYPE_BUL,
+                                                XUD_EPTYPE_BUL};
+
```

Comment:
Quite a few main.xc files `#include "test_main.xc"` and use the common facilities it provides.  THis one does not.  Is that intentional or an oversight?

Reply:
Test removed.